mailu/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-appVersion: 2024.06.41
-version: 2.2.2
+appVersion: 2.0.30
+version: 1.5.1
```

Comment:
Please don't bump the version of the chart in the PR. There is a GitHub workflow for it